samples/tab-calendar/ts/src/components/Tab.tsx

```diff
@@ -0,0 +1,380 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import React from "react";
+import "./App.css";
+import * as microsoftTeams from "@microsoft/teams-js";
+/**
+ * This tab component renders the main tab content
+ * of your app.
+ */
+
+export interface IAllItemsDetail {
+  id: string;
+  subject: string;
+}
+export interface ITabProps {}
+interface ITabState {
+  context?: microsoftTeams.app.Context;
+  ssoToken: string;
+  consentRequired: boolean;
+  consentProvided: boolean;
+  graphAccessToken: string;
+  error: boolean;
+  stateDatetime: string;
+  endDatetime: string;
+  attendees: string;
+  subject: string;
+  content: string;
+  allItems: IAllItemsDetail[];
+}
+class Tab extends React.Component<ITabProps, ITabState> {
+  constructor(props: ITabProps) {
+    super(props);
+    this.state = {
+      context: undefined,
+      ssoToken: "",
+      consentRequired: false,
+      consentProvided: false,
+      graphAccessToken: "",
+      error: false,
+      stateDatetime: "",
+      endDatetime: "",
+      attendees: "",
+      subject: "",
+      content: "",
+      allItems: [],
+    };
+
+    //Bind any functions that need to be passed as callbacks or used to React components
+    this.attendeesHandleChange = this.attendeesHandleChange.bind(this);
+    this.composeMeeting = this.composeMeeting.bind(this);
+    this.startHandleChange = this.startHandleChange.bind(this);
+    this.endHandleChange = this.endHandleChange.bind(this);
+    this.subjectHandleChange = this.subjectHandleChange.bind(this);
+    this.contentHandleChange = this.contentHandleChange.bind(this);
+
+    this.consentSuccess = this.consentSuccess.bind(this);
+    this.consentFailure = this.consentFailure.bind(this);
+    this.unhandledFetchError = this.unhandledFetchError.bind(this);
+    this.callGraphFromClient = this.callGraphFromClient.bind(this);
+    this.showConsentDialog = this.showConsentDialog.bind(this);
+  }
+
+  //React lifecycle method that gets called once a component has finished mounting
+  //Learn more: https://reactjs.org/docs/react-component.html#componentdidmount
```

Comment:
// React lifecycle method that gets called once a component has finished mounting
// Learn more: https://reactjs.org/docs/react-component.html#componentdidmount

Reply:
Fixed